include/physics/systems/physicssystem.hpp

```diff
@@ -253,10 +251,23 @@ namespace legion::physics
             {
                 for (auto& contact : manifold.contacts)
                 {
-                    PhysicsSystem::contactPoints.push_back(contact);
+                    contact.refCollider->converganceIdentifiers.clear();
                 }
             }
 
+            for (auto& manifold : manifoldsToSolve)
+            {
+                for (auto& contact : manifold.contacts)
+                {
+                    //assert(contact.label.IsSet());
+
+                    contact.refCollider->AddConverganceIdentifier(contact);
```

Comment:
Maybe this can be moved to the previous dubble for-loop

Reply:
now that i look at it i just realized why this is necessary: when AddConverganceIdentifier is called and it
ads to the std::vector converganceIdentifiers. This is why it needs to be cleared first